mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/TypeUtils.java

```diff
@@ -111,6 +114,16 @@ public static ExpressionSupport getExpressionSupport(MetadataType metadataType)
         .orElse(ExpressionSupport.SUPPORTED);
   }
 
+  public static ParameterRole getParameterRole(MetadataType metadataType) {
```

Comment:
because an operation can have a parameter which is a pojo which is not the content itself, but contains fields which are content. Example: HttpRequestBuilder, HttpResponseBuilder, EmailBuilder, etc